components/molecules/Greeting.js

```diff
@@ -2,7 +2,7 @@ import { Message, Heading } from '@dts-stn/service-canada-design-system'
 
 export default function Greeting(props) {
   return (
-    <div>
+    <div id="mainContent">
```

Comment:
Shouldn't this be added to the Layout component so it's on every page?

Reply:
Maybe, I wasn't sure